src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -129,6 +130,19 @@ public void clear() {
         internalList.clear();
     }
 
+
+    /**
+     * Sorts the list of persons in alphabetical order.
+     */
+    public void sort() {
+        Comparator<Person> personNameComparator = (personA, personB) -> {
+            Name personAName = personA.getName();
+            Name personBName = personB.getName();
+            return personAName.compareTo(personBName);
+        };
```

Comment:
if `personAName == "alice"` and `personBName == "Bob"`, "Bob" will be put before "alice". Is this a desired behavior?